src/clr-angular/data/datagrid/datagrid-action-overflow.spec.ts

```diff
@@ -47,7 +54,7 @@ export default function(): void {
       expect(context.clarityDirective.open).toBe(false);
     });
 
-    it('closes the menu when clicked outside of the host', () => {
+    it('closes the menu when clicked outside of the host', function() {
```

Comment:
Not blocking the PR on this but seems a bit messy to have a mixture of syntax (i.e. `function() { ... }` and `() => { ... }`) in this spec file. Would be nice to use one or the other at least on a per file basis.

Reply:
I'll try to align.

Reply:
Updated the lambda functions to use `function () {}` instead of `() => {}`